Source/JavaScriptCore/runtime/LiteralParser.cpp

```diff
@@ -165,16 +165,6 @@ ALWAYS_INLINE JSString* LiteralParser<CharType>::makeJSString(VM& vm, typename L
     return jsString(vm, Identifier::fromString(vm, token->stringToken16, token->stringLength).string());
 }
 
-static ALWAYS_INLINE bool cannotBeIdentPartOrEscapeStart(LChar)
-{
-    RELEASE_ASSERT_NOT_REACHED();
-}
-
-static ALWAYS_INLINE bool cannotBeIdentPartOrEscapeStart(UChar)
-{
-    RELEASE_ASSERT_NOT_REACHED();
-}
-
```

Comment:
This was added in https://github.com/WebKit/WebKit/commit/6338e2ba60d88e04ffed6703dfa1028a611fd6dc to remove a warning so not sure if this is going to become an issue